dev.corusoft.eticketia.domain.exceptions.DomainException=error de ejecución interno
# ************************* AUTH ************************* #
dev.corusoft.eticketia.domain.exceptions.auth.UserAlreadyExistsException=usuario con ID {0} ya existe
dev.corusoft.eticketia.domain.exceptions.auth.EmailAlreadyRegisteredException=el email {0} ya está registrado
dev.corusoft.eticketia.domain.exceptions.auth.UserNotFoundException=usuario con ID {0} no encontrado
# ************************* TICKET ************************* #
dev.corusoft.eticketia.domain.exceptions.ticket.UnableToParseTicketException=error extrayendo contenido del ticket
